include/task.h

```diff
@@ -147,6 +147,26 @@ typedef struct xTASK_STATUS
 	configSTACK_DEPTH_TYPE usStackHighWaterMark;	/* The minimum amount of stack space that has remained for the task since the task was created.  The closer this value is to zero the closer the task has come to overflowing its stack. */
 } TaskStatus_t;
 
+/* Possibe return values for vTaskGetCurrentBlocker() */
+typedef enum
+{
+    eBlockedForEvent = 0,
+    eBlockedForNotification,
+    eBlockedForTime,
+    eNotBlocked
+} eBlockedStatus;
+
+/* Used with vTaskGetCurrentBlocker to return details on what is blocking a task */

```

Comment:
It may be necessary to update some of these comments so they are complete sentences